ui/admin/app/templates/scopes/scope/credential-stores/credential-store/worker-filter.hbs

```diff
@@ -0,0 +1,51 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{page-title (t 'resources.credential-store.titles.worker-filter')}}
+
+<Rose::Layout::Page as |page|>
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+
+  <page.header>
+    <CredentialStores::CredentialStore::Header @model={{@model}} />
+  </page.header>
+
+  <page.navigation>
+    <CredentialStores::CredentialStore::Nav @model={{@model}} />
+  </page.navigation>
+
+  <page.actions>
+    <CredentialStores::CredentialStore::Actions
+      @model={{@model}}
+      @delete={{fn this.credentialStores.delete @model}}
+    />
+  </page.actions>
+
+  <page.body>
```

Comment:
Related to the Hds::ApplicationState component. It doesn't have margin-top so we need to add it. We typically add a class to the `page.body` and add custom css for that. Here is an example of that:
```
.role-grant-scopes {
  .hds-application-state {
    margin-top: 3rem;
  }
}
```